docs/en/setup/backend/backend-k8s-monitoring-cilium.md

```diff
@@ -0,0 +1,107 @@
+# Kubernetes (K8s) monitoring from Rover
+
+SkyWalking uses the Cilium Fetcher to gather traffic data between services from Cilium Hubble via the Observe API. It then leverages the [OAL System](./../../concepts-and-designs/oal.md) for metrics and entity analysis.
+
+## Data flow
+
+SkyWalking Fetch Cilium Node and Observe Data from gRPC API, analysis to generate entity and using [OAL](./../../concepts-and-designs/oal.md) to generating metrics.
+
+## API Requirements
+
+1. [Peers API](https://github.com/cilium/cilium/blob/main/api/v1/peer/peer_grpc.pb.go#L33-L39): Listen the hubble node in the cluster, OAP would communicate with Hubble node to obtain Observe data.
+2. [Observe API](https://github.com/cilium/cilium/blob/main/api/v1/observer/observer_grpc.pb.go#L41): Fetch the Flow data from Hubble node.
+
+## Setup
+1. Please following the [Setup Hubble Observability documentation](https://docs.cilium.io/en/stable/gettingstarted/hubble_setup/) to setting the Hubble for provided API.
+2. Setup cilium receiver module by the following configuration.
+```yaml
+cilium-fetcher:
+    selector: ${SW_CILIUM_FETCHER:default}
+    default:
+        peerHost: ${SW_CILIUM_FETCHER_PEER_HOST:hubble-peer.kube-system.svc.cluster.local}
+        peerPort: ${SW_CILIUM_FETCHER_PEER_PORT:80}
```

Comment:
```suggestion
        # Host name and port of Hubble server
        peerHost: ${SW_CILIUM_FETCHER_PEER_HOST:hubble-peer.kube-system.svc.cluster.local}
        peerPort: ${SW_CILIUM_FETCHER_PEER_PORT:80}
```

What is the `peer`? Is there a component called Hubble server or something?

Reply:
peer is a service of the Hubble. the cilium fetcher needs getting all cilium nodes through this service. 

```shell
$kubectl get services -A
NAMESPACE      NAME                            TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)               AGE
default        details                         ClusterIP   34.118.228.1     <none>        9080/TCP              42h
default        kubernetes                      ClusterIP   34.118.224.1     <none>        443/TCP               42h
default        productpage                     ClusterIP   34.118.235.150   <none>        9080/TCP              42h
default        ratings                         ClusterIP   34.118.239.54    <none>        9080/TCP              42h
default        reviews                         ClusterIP   34.118.229.30    <none>        9080/TCP              42h
gmp-system     alertmanager                    ClusterIP   None             <none>        9093/TCP              42h
gmp-system     gmp-operator                    ClusterIP   34.118.231.151   <none>        8443/TCP,443/TCP      42h
istio-system   elasticsearch-master            ClusterIP   34.118.234.99    <none>        9200/TCP,9300/TCP     21h
istio-system   elasticsearch-master-headless   ClusterIP   None             <none>        9200/TCP,9300/TCP     21h
istio-system   skywalking-oap                  ClusterIP   34.118.228.217   <none>        11800/TCP,12800/TCP   21h
istio-system   skywalking-ui                   ClusterIP   34.118.226.129   <none>        80/TCP                21h
kube-system    default-http-backend            NodePort    34.118.233.215   <none>        80:31226/TCP          42h
kube-system    hubble-peer                     ClusterIP   34.118.232.156   <none>        443/TCP               42h
kube-system    hubble-relay                    ClusterIP   34.118.233.50    <none>        80/TCP                178m
kube-system    kube-dns                        ClusterIP   34.118.224.10    <none>        53/UDP,53/TCP         42h
kube-system    metrics-server                  ClusterIP   34.118.228.67    <none>        443/TCP               42h
```